package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "6.10.7-case-flags-v2-welsh-language-support-additional-rework-2",
+  "version": "6.16.0-query-management-write-doc-view-v10-fix",
```

Comment:
Again, version had gone backwards - if going to Release we need to be consistent with Release